### ISSUE TITLE
Reduced glutamate in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6035]
**Preferred term label:** Reduced glutamate in urine

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of glutamate in the urine is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormality of urine homeostasis HP:0003110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
reduced amino acid levels in urine are not medically relevant
